include/wx/richmsgdlg.h

```diff
@@ -46,13 +47,25 @@ class WXDLLIMPEXP_CORE wxRichMessageDialogBase : public wxGenericMessageDialog
 
     virtual bool IsCheckBoxChecked() const { return m_checkBoxValue; }
 
+    void SetFooterText(const wxString& footerText)
+        { m_footerText = footerText; }
+
+    wxString GetFooterText() const { return m_footerText; }
+
+    void SetFooterIcon(long icon)
+        { m_footerIcon = icon; }
+
+    long GetFooterIcon() const { return m_footerIcon; }
```

Comment:
It would be really nice to have an enum for these values instead of using long one day (why `long` and not `int` BTW?)...